docs/build.md

```diff
@@ -0,0 +1,40 @@
+# Bloc-DID-Method - Build
+
+## Prerequisites
+- Go 1.13
+- Make
+
+## Developer Setup
+
+In order to access build images you have to create a [personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) in your github account, with the read:packages and repo permissions.
+
+Then, using your token, run the following command:
+
+    docker login -u <username> -p <github token with read:packages and repo permission> docker.pkg.github.com
```

Comment:
I feel better to give just the reference to github help page. Similar to the doc in aries-framework-go https://github.com/hyperledger/aries-framework-go/blob/master/docs/test/build.md#prerequisites-for-running-tests-and-demos

Reply:
done